ironic/templates/configmaps/ironic-pxe.sh.yaml.j2

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  ironic-pxe.sh: |+
+    #!/bin/bash
+    set -ex
+
+    for pxe_file in /var/lib/tftpboot/pxelinux.0 /var/lib/tftpboot/chain.c32 /usr/lib/syslinux/pxelinux.0 \
+                    /usr/lib/syslinux/chain.c32 /usr/lib/PXELINUX/pxelinux.0 \
+                    /usr/lib/syslinux/modules/bios/chain.c32; do
+        if [[ -e "$pxe_file" ]]; then
+            cp "$pxe_file" /tftpboot
+        fi
+    done
+
+    mkdir -p /tftpboot/pxelinux.cfg
+    echo "default menu.c32" > /tftpboot/pxelinux.cfg/default
```

Comment:
I guess that it would be much more readable if you merge it into one echo command.